### ISSUE TITLE
Run DCE rake task for Oxford yearbooks

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-curate#834]
Run the DCE pre-processor rake task for Yearbooks on the Oxford yearbooks CSV. This collection uses the LIMB output.

[DCE rake task documentation](https://wiki.service.emory.edu/pages/viewpage.action?pageId=161546595)

- [x] Pre-processed [CSV for Oxford LIMB yearbooks](https://github.com/emory-libraries/cor-data-transfer/blob/master/curate-ingests/2019/in_development/yearbooks/Yearbooks-Oxford-Curate-LIMB.csv)
- [x] [Alma records for yearbooks](https://github.com/emory-libraries/cor-data-transfer/blob/master/curate-ingests/2019/in_development/yearbooks/alma-yearbooks.xml)
- [x] Transfer the processed CSV file to the cor-data-transfer repo: https://github.com/emory-libraries/cor-data-transfer/tree/master/curate-ingests/2019/processed/yearbooks

**Comment by devanshu-m** [comment]
@eporter23 is this still blocked by another ticket?
